docs/guide/schemagen.md

```diff
@@ -43,33 +45,30 @@ feature {
 ...
 ```
 
-The following TFX libraries use the schema:
+The following TFX libraries use the schema generated by `SchemaGen`:
 
 *   TensorFlow Data Validation
 *   TensorFlow Transform
 *   TensorFlow Model Analysis
 
-In a typical TFX pipeline SchemaGen generates a schema, which is consumed by the
+In a typical TFX pipeline, SchemaGen generates a schema which is consumed by the
 other pipeline components.
 
-Note: The auto-generated schema is best-effort and only tries to infer basic
-properties of the data. It is expected that developers review and modify it as
-needed.
 
 ## SchemaGen and TensorFlow Data Validation
 
-SchemaGen makes extensive use of [TensorFlow Data Validation](tfdv.md) for inferring a schema.
+SchemaGen makes extensive use of the [TensorFlow Data Validation](tfdv.md) library for inferring a schema
 
 ## Using the SchemaGen Component
 
-A SchemaGen pipeline component is typically very easy to deploy and requires little
+A SchemaGen component is typically very easy to deploy and requires little
 customization. Typical code looks like this:
 
 ```python
-from tfx import components
+from tfx.components import SchemaGen
```

Comment:
The reason for current way of writing this can be found at [Google python style guide](http://google.github.io/styleguide/pyguide.html#224-decision)

For most cases we import modules, but not classes.

Reply:
Thanks for correcting me, @numerology. Will make changes to the RC PR of this doc too. I used "from tfx.components import <component_name> in my updates to the StatisticsGen docs. However, I missed out here.